rtsp_test.go

```diff
@@ -94,7 +94,8 @@ func TestRTSPCamera(t *testing.T) {
 				timeoutCtx, timeoutCancel := context.WithTimeout(context.Background(), time.Second*10)
 				defer timeoutCancel()
 				config := resource.NewEmptyConfig(camera.Named("foo"), ModelAgnostic)
-				config.ConvertedAttributes = &Config{Address: "rtsp://" + h.s.RTSPAddress, RTPPassthrough: true}
+				shouldPassthrough := true
+				config.ConvertedAttributes = &Config{Address: "rtsp://" + h.s.RTSPAddress, RTPPassthrough: &shouldPassthrough}
```

Comment:
we need a test that shows that `&Config{Address: "rtsp://" + h.s.RTSPAddress}` results in `when RTPPassthrough = true` behavior.

Reply:
Added nil case, true case, false case in a loop